DEVELOPERS.md

```diff
@@ -92,12 +92,31 @@ podman-compose -f compose.dev.yml exec app npm run watch
 
 This will re-build the bundled JS files any time you save a change to a `.js` file which is useful while you're writing Javascript. This command does not exit until you press CTRL-C, so make sure to run it in a separate terminal.
 
+# Virtual Environment and Poetry Setup
+Dependencies for testing and building documentation are managed with [Poetry](https://python-poetry.org/) within a virtual environment. To create a virtual environment and install Poetry for testing and building documentation, run the following commands:
+
+On Windows:
+```shell
+python -m venv env
+.\env\Scripts\Activate.ps1
+pip install poetry
+```
+
+On MacOS/Linux:
+```shell
+python3 -m venv env
+source env/bin/activate
+pip install poetry
+```
```

Comment:
In the [Poetry installation documentation](https://python-poetry.org/docs/#installation), there's a big warning that poetry needs to be installed in its own environment, separate from the dependencies it manages. The reasoning is that `poetry install` might install an incompatible version of a package that it itself depends on.

The recommended method to install Poetry is with `pipx`.

I would recommend changing this documentation to:

---

# Virtual Environment and Poetry Setup

Python dependencies are managed with [Poetry](https://python-poetry.org/). To install Poetry and create a virtual environment with the minimally necessary packages, follow these instructions:

First, install `pipx` via the [official pipx installation instructions](https://pipx.pypa.io/stable/installation/).

After installing `pipx`, install [Poetry](https://python-poetry.org/):

```shell
pipx install poetry
```

Next, create an activate a virtual environment from the root of the repository:

```shell
# If on Windows:
python -m venv env
.\env\Scripts\Activate.ps1

# If on MacOS or Linux
python3 -m venv env
source env/bin/activate
```

Finally, install the dependencies with `poetry`:

```shell
poetry install
```

Reply:
> In the [Poetry installation documentation](https://python-poetry.org/docs/#installation), there's a big warning that poetry needs to be installed in its own environment, separate from the dependencies it manages. The reasoning is that poetry install might install an incompatible version of a package that it itself depends on.

Ah I misunderstood it for "Install everything in a dedicated virtual environment". This makes a lot of sense.

Resolved by: https://github.com/NationalCentreTruthReconciliation/Secure-Record-Transfer/pull/269/commits/efcda9b6e8aeaca65c9c51fd3bfdd16774362412